src/main/kotlin/org/opensearch/replication/ReplicationPlugin.kt

```diff
@@ -189,6 +189,8 @@ internal class ReplicationPlugin : Plugin(), ActionPlugin, PersistentTaskPlugin,
             Setting.Property.IndexScope, Setting.Property.Dynamic)
         val REPLICATION_INDEX_TRANSLOG_RETENTION_SIZE: Setting<ByteSizeValue> = Setting.byteSizeSetting("index.plugins.replication.translog.retention_size",
             ByteSizeValue(512, ByteSizeUnit.MB), Setting.Property.Dynamic, Setting.Property.IndexScope)
+        val REPLICATION_FOLLOWER_BLOCK_START: Setting<Boolean> = Setting.boolSetting("plugins.replication.follower.block.start", false,
+                Setting.Property.Dynamic, Setting.Property.NodeScope)
     }
```

Comment:
What is the purpose of this setting?

Reply:
This can be used for the domains during upgrade with existing cross cluster connections.